dataclients/kubernetes/kube.go

```diff
@@ -88,7 +88,7 @@ type Options struct {
 	// in the cluster-scope.
 	KubernetesNamespace string
 
-	// KubernetesIngressV1 is used to switch between v1beta1 and v1. Kubernetes version 1.22 stopped support
+	// *DEPRECATED* KubernetesIngressV1 is used to switch between v1beta1 and v1. Kubernetes version 1.22 stopped support
```

Comment:
minor: I think there is some kind of convention that e.g. staticcheck understands, maybe a `// DEPRECATED` comment on the same line.

Reply:
I used it because we already use it like this, but great to know, maybe we should change this everywhere at once.